dio/lib/src/dio_exception.dart

```diff
@@ -154,7 +165,8 @@ class DioException implements Exception {
   }) =>
       DioException(
         type: DioExceptionType.connectionError,
-        message: 'The connection errored: $reason',
+        message: 'The connection errored: $reason. '
+            'This indicates an error which most likely can\'t be solved with DIO.',
```

Comment:
Correct me if I'm wrong, I'm not sure about *with* or *by*

```suggestion
        message: 'The connection errored: $reason. '
            'This indicates an error which most likely cannot be solved by the library.',
```

Reply:
> I'm not sure about with or by

Me neither, but `by` reads better, so I go with it.